File: nlp/toxic_bert_api.js
async function query(data) {
	const response = await fetch(
		"https://api-inference.huggingface.co/models/unitary/toxic-bert",
		{
			headers: { Authorization: "Bearer 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" },
			method: "POST",
			body: JSON.stringify(data),
		}
	);
	const result = await response.json();
	return result;
}

query({"inputs": "I like you. I love you"}).then((response) => {
	console.log(JSON.stringify(response));
});

File: nlp/toxic_bert_api.py
import requests
API_KEY = "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"

API_URL = "https://api-inference.huggingface.co/models/unitary/toxic-bert"
headers = {"Authorization": f"Bearer {API_KEY}"}

def query(payload):
	response = requests.post(API_URL, headers=headers, json=payload)
	return response.json()
#input text	
text = "This is a great initiative!"

output = query({
	"inputs": text,
})

print(output)
